code/modules/projectiles/guns/energy/energy_gun.dm

```diff
@@ -25,6 +25,20 @@
 	can_flashlight = FALSE // Can't attach or detach the flashlight, and override it's icon update
 	weapon_weight = WEAPON_LIGHT
 
+/obj/item/gun/energy/e_gun/mini/heads
+	name = "PTSD"
```

Comment:
I think it should be named something more suitable, like the full name 'Personal Tiny Self Defense' or just Personal Energy Gun or something

Reply:
![image](https://user-images.githubusercontent.com/9547572/168460235-1a6d808f-ff11-42e3-9d47-e23b29fa3923.png)